composer/datasets/synthetic.py

```diff
@@ -100,9 +100,11 @@ def __getitem__(self, idx: int):
                 input_target[:, 0] = 1.0
             elif self.label_type == SyntheticDataLabelType.CLASSIFICATION_INT:
                 assert self.num_classes is not None
-                input_target = torch.randint(0,
-                                             self.num_classes, (self.num_unique_samples_to_create,),
-                                             device=self.device)
+                if self.label_shape:
```

Comment:
I added this for synthetic segmentation datasets (i.e. image and labels are the same dimension). Is this okay? Also, is RANDOM_INT setting below still relevant?

Reply:
Sure, cool with this so long as it doesn't break the existing tests. Hmm yeah, nothing really uses random_int. I guess for consistency, could you also copy this change down there? Or remove the `RANDOM_INT` option for synthetic data labels?

Reply:
`RANDOM_INT` seemed redundant, so I removed it